girder/utility/assetstore_utilities.py

```diff
@@ -24,6 +24,11 @@
 from girder import events
 
 
+assetstoreTable = {AssetstoreType.FILESYSTEM: FilesystemAssetstoreAdapter,
```

Comment:
If we're going to have an 80-character width limit, then we really have to globally adopt the dict style rule expressed by @fogleman . This should indeed be changed to that.